### PR TITLE
Adds a new option to be able to specify php.ini file

### PR DESCRIPTION
**Pull request by falahati**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,6 +77,11 @@
           "default": null,
           "description": "Specify the location of your PHP executable. If empty, the extension will use the PHP executable which appears in the $PATH environment variable"
         },
+        "phpserver.phpConfigPath": {
+          "type": "string",
+          "default": null,
+          "description": "Specify the location of your php.ini file. If empty, the extension lets the PHP executable to decides."
```
Comment (brapifra) [inline package.json]:
```suggestion
          "description": "Specify the location of your php.ini file. If empty, the extension uses the default path."
```

**Comment by brapifra** [comment]
Hi @falahati 
Thank you for your contribution! I'll be happy to merge it after you make the changes I suggested :)

**Comment by falahati** [comment]
@brapifra, changes are made.

**Comment by josematheus10** [comment]
Do you support https?

**Comment by brapifra** [comment]
So sorry I've left this open @falahati 
I'm going to merge it soon. Thank you

**Pull request merged**